easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-1.7-GCC-4.6.3-hwloc-chkpt.eb

```diff
@@ -23,7 +23,10 @@ dependencies = [('hwloc', '1.5.1')]
 
 # enable checkpointing support
 withchkpt = True
-osdependencies = ['blcr-libs', 'blcr-devel']
+osdependencies = [
+    ('libcr-dev', 'blcr-devel'),
```

Comment:
Yes, it is, cfr. http://rpm.pbone.net/index.php3/stat/4/idpl/20828347/dir//com/blcr-devel-0.8.1-1.fc10.x86_64.rpm.html

So, fine.